package.json

```diff
@@ -14,52 +15,52 @@
     "@gravity-ui/components": "^2.9.1",
     "@gravity-ui/date-utils": "^1.1.1",
     "@gravity-ui/i18n": "^1.0.0",
+    "@gravity-ui/icons": "^2.8.1",
     "@gravity-ui/navigation": "^1.8.0",
     "@gravity-ui/paranoid": "^1.4.0",
     "@gravity-ui/react-data-table": "^1.0.3",
-    "@types/numeral": "^2.0.2",
-    "@types/qs": "^6.9.7",
-    "@types/react": "^17.0.58",
-    "axios": "0.19.2",
-    "bem-cn-lite": "4.0.0",
+    "@gravity-ui/uikit": "^5.24.0",
+    "axios": "^0.21.2",
+    "bem-cn-lite": "^4.1.0",
     "copy-to-clipboard": "^3.3.3",
     "crc-32": "^1.2.2",
-    "history": "4.10.1",
-    "js-cookie": "2.2.1",
-    "lodash": "4.17.11",
-    "monaco-editor": "0.24.0",
-    "numeral": "2.0.6",
-    "path-to-regexp": "3.0.0",
+    "history": "^4.10.1",
+    "lodash": "^4.17.11",
+    "monaco-editor": "^0.31.1",
+    "numeral": "^2.0.6",
+    "path-to-regexp": "^3.0.0",
     "qs": "^6.11.0",
     "react-error-boundary": "^4.0.12",
-    "react-json-inspector": "7.1.1",
-    "react-list": "0.8.11",
-    "react-monaco-editor": "0.30.1",
-    "react-redux": "7.2.6",
-    "react-router": "5.2.1",
-    "react-router-dom": "5.3.0",
-    "react-scripts": "5.0.1",
-    "react-split": "2.0.14",
-    "redux": "4.0.1",
-    "redux-location-state": "2.6.0",
-    "redux-thunk": "2.3.0",
-    "reselect": "4.1.6",
-    "sass": "1.32.8",
+    "react-json-inspector": "^7.1.1",
+    "react-list": "^0.8.11",
+    "react-monaco-editor": "^0.47.0",
+    "react-redux": "^7.2.6",
+    "react-router": "^5.2.1",
+    "react-router-dom": "^5.3.0",
+    "react-split": "^2.0.14",
+    "redux": "^4.0.1",
+    "redux-location-state": "^2.6.0",
+    "redux-thunk": "^2.3.0",
+    "reselect": "^4.1.6",
     "url": "^0.11.0",
     "use-query-params": "^2.2.1",
-    "web-vitals": "1.1.2",
+    "web-vitals": "^1.1.2",
     "ydb-ui-components": "^3.6.0"
   },
   "scripts": {
     "start": "react-app-rewired start",
-    "dev": "DISABLE_ESLINT_PLUGIN=true TSC_COMPILE_ON_ERROR=true REACT_APP_BACKEND=http://localhost:8765 npm start",
+    "dev": "DISABLE_ESLINT_PLUGIN=true TSC_COMPILE_ON_ERROR=true REACT_APP_BACKEND=http://localhost:8765 REACT_APP_META_BACKEND=undefined npm start",
```

Comment:
WIth this command a request to `http://localhost:3000/cluster/undefined/meta/clusters` is sent

<img width="1728" alt="Screen Shot 2024-02-09 at 12 49 59" src="https://github.com/ydb-platform/ydb-embedded-ui/assets/67755036/0893f523-5565-4160-a5ec-94e1552f4472">

Reply:
fixed